src/components/button.css

```diff
@@ -11,7 +11,7 @@
   font: normal 700 1.255rem/3.75rem Lato;
   color: var(--clr-white);
   cursor: pointer;
-  margin: 5px;
+  margin-right: 15px;
```

Comment:
Just want to check if this will not change the styling of the button on the header slide?